src/components/header/Header.scss

```diff
@@ -63,6 +63,7 @@
 .header .logo-name {
   font-family: "Agustina Regular";
   font-weight: bold;
+  font-feature-settings: "liga","clig off";
```

Comment:
What does this do?

Reply:
Those are the original settings of the font, seems to fix the logo issue kart chick was talking about. I think someone removed it with previous PR